documentation/release_notes.markdown

```diff
@@ -12,6 +12,14 @@ canonical: "/puppetserver/latest/release_notes.html"
 
 For release notes on versions of Puppet Server prior to Puppet Server 5, see [docs.puppet.com](https://docs.puppet.com/puppetserver/2.8/release_notes.html).
 
+## Puppet Server 5.3.12
+
+Released 10 March  2020
+
+### Resolved issue 
+
+To prevent information exposure as a result of [CVE-2020-7943](https://puppet.com/security/cve/CVE-2020-7943), the `/metrics/v1` endpoints are disabled by default, and access to the `/metrics/v2` endpoints are restricted to localhost. [PE-28468](https://tickets.puppetlabs.com/browse/PE-28468). 
```

Comment:
I don't see any precedence in the release notes for linking to private JIRA projects from a public project like this; I'm not sure if we should?

Reply:
oh yeah, good call

Reply:
They usually link to SEVER tickets, but maybe PE is all private? If so I'll remove. Thanks!

Reply:
yes, @clairecadman, PE is a private project.

Reply:
I've updated this and also 6.9.1. Thanks!